wit/readwrite.wit

```diff
@@ -1,26 +1,56 @@
 /// A keyvalue interface that provides simple read and write operations.
+/// 
+/// A read/write operation is an operation that acts on a single key-value pair.
+/// 
+/// The value in the key-value pair is defined as a `u8` byte array and the intention
```

Comment:
This comment is added after all the code reviews to clarify why is the value in key-value pair is defined as a `list<u8>`.